cmd/promtool/tsdb.go

```diff
@@ -624,3 +622,18 @@ func checkErr(err error) int {
 	}
 	return 0
 }
+
+func readOpenMetrics(path string, outputDir string) (err error) {
+	input, err := os.Open(path)
+	if path != "" {
```

Comment:
Why do you have this check?

Reply:
I had it for the previous implementation. Forgot to remove it. Removing.